test/controllers/users_controller_test.rb

```diff
@@ -8,20 +8,20 @@ def setup
   end
 
   def test_user_creation
-    field_value  = { name: Faker::Name.name[1..18] , mobile_no: Faker::Number.number(digits: 10) , email: Faker::Internet.email , password:"Abcdef@1" , address:Faker::Lorem.paragraph}
+    field_value  = { name: Faker::Name.name[1..18] , mobile_no: Faker::Number.number(digits: 10) , email: Faker::Internet.email , password:@user.password , address:Faker::Lorem.paragraph}
```

Comment:
please check the spacing 

addres**s: F**aker::Lorem.paragraph}

Reply:
changed